File: Haskell/beginner/1095.hs
-- https://judge.beecrowd.com/pt/problems/view/1095
-- Sequência IJ 1


-- Recursive
sequenceIJ :: Int -> Int -> IO ()
sequenceIJ i j
    | j < 0 = return ()
    | otherwise = do
        putStrLn $ "I=" ++ show i ++ " J=" ++ show j
        sequenceIJ (i + 3) (j - 5)

-- Main function

main = sequenceIJ 1 60

-- By Fernando Serra
-- https://github.com/fernandosserra

File: C/beginner/1095.c
// https://judge.beecrowd.com/pt/problems/view/1095
// Sequência IJ 1

// Libraries
#include <stdio.h>

// Main Function
int main(){
    // Starting Code
    int i = 1, j = 60;
    
    // Iteration
    while (j >= 0)
    {
        // Printing
        printf("I=%d J=%d\n", i, j);
        i += 3;
        j -= 5;
    }
    // Ending Code
    return 0;
}

// By Fernando Serra
// https://github.com/fernandosserra